src/components/organisms/UiModal/UiModal.vue

```diff
@@ -368,6 +368,10 @@ export interface ModalProps {
    * Use this props to pass attrs for close UiIcon
    */
   iconCloseAttrs?: IconAttrsProps;
+  /**
```

Comment:
I added` slot#content` because `slot#description` is dependent on the `props.title`. 
What if, for some reason, the `props.title` is an empty string? The description won't appear, and there will be a modal with only buttons.
If you think I should remove this slot and use the `description`, I'll change the code, but are you sure that's what we want? Or maybe I can change the condition:
`const hasDescription = computed(() => !!(props.title && props.description));`
to depend only on` props.description`?

Reply:
`slots.desctiption` or `slots.title` should show you shen user use slots to put the content :) maybe we can use this knowledge to make some condition WDYT?